controllers/roverController.ts

```
@@ -0,0 +1,17 @@
+import { Request, Response } from "express"
+import nasaClient from "../clients/nasaClient"
+import Rover from "../models/roverData"
+
+export const RoverController = {
+    Index: async (req: Request, res: Response) => {
+        try {
+            const roverData: Rover[] | void = await nasaClient.getRoverList()
+            const roverNames: string[] = []
+            roverData!.forEach((rover: Rover) => roverNames.push(rover.name))
```

Biome

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)